src/util.js

```diff
@@ -0,0 +1,9 @@
+
+function isValidUUID(input) {
```

Comment:
I think a better naming for this would probably be `isValidUuid`, seeing UUID as a single "word". That's how we generally consistently do it. Otherwise we'll end up with names like `XMLHttpRequest` (real example).